package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "luxon": "^3.3.0",
```

Comment:
Great pick!! This package has no dependencies and a ton of downloads - good 👁️ AS!